spec/services/file_attachment_blob_service_spec.rb

```diff
@@ -2,50 +2,45 @@
 
 RSpec.describe FileAttachmentBlobService do
   let(:file) { fixture_file_upload("files/text-file.txt") }
-  let(:revision) do
-    build(:revision, file_attachment_revisions: [file_attachment_revision])
-  end
-  let(:file_attachment_revision) do
-    build(:file_attachment_revision, filename: "text-file.txt")
-  end
+  let(:revision) { build(:revision) }
+  let(:user) { build(:user) }
+  let(:instance) { FileAttachmentBlobService.new(revision, user) }
```

Comment:
This reminds me of `subject` ;-).

Reply:
haha - but the subject is the method. I know, I know - let's not go there again.